packages/devtools-reps/src/launchpad/index.js

```diff
@@ -31,6 +35,10 @@ function onConnect(connection) {
     },
 
     getObjectClient: function (grip) {
+      if (nodeIsLongString(grip)) {
```

Comment:
i'm not sure we should use nodeIsLongString here. We are dealing with a grip, and nodeIsLongString expect a ObjectInspector node, so I'm unsure this is correct.
We could use https://github.com/devtools-html/devtools-core/blob/b6819f649da535d42005ee06403bd44efe38633a/packages/devtools-reps/src/reps/string.js#L270-L272 which seems like a safe bet here.